fre/cmor/cmor_mixer.py

```diff
@@ -1,408 +1,622 @@
 #!/usr/bin/env python
```

Comment:
i'm tempted to remove this because it could unintentionally point someone to a python where things *kinda* work, but don't actually. it hardly seems necessary for functioning anyways.

either it points to a `python2` nearby, and things should error quickly. Or it points to a `python3` nearby with just enough bells and whistles to not fail immediately and clearly